packages/components/switch.tsx

```diff
@@ -0,0 +1,99 @@
+import { connect, styled, warn } from "frontity";
+import { Package } from "frontity/types";
+import React, { isValidElement, ReactNodeArray } from "react";
+
+const description404 = (
+  <>
+    That page can’t be found{" "}
+    <span role="img" aria-label="confused face">
+      😕
+    </span>
+  </>
+);
+
+const description = (
+  <>
+    Don&apos;t panic! Seems like you encountered an error. If this persists,
+    <a href="https://community.frontity.org"> let us know </a> or try refreshing
+    your browser.
+  </>
+);
+
+// Default 404 Error Component
+const DefaultErrorComponent = ({ state }) => {
```

Comment:
Hey, I would remove this `DefaultErrorComponent`, I guess it's better for the `Switch` component to render nothing (return `null`) if there is no default component among its children, right? Just my opinion.